frame/dapps-staking/src/lib.rs

```diff
@@ -494,6 +496,8 @@ pub enum RewardDestination {
     /// Rewards are transferred to stakers balance and are immediately re-staked
     /// on the contract from which the reward was received.
     StakeBalance,
+    /// Rewards are deposited into a delegated beneficiary account.
+    BeneficiaryBalance,
```

Comment:
Yes, I did put a new type in the enum `RewardDestination`  for specifying that it has been sent to the beneficiary balance. But for simplicity purposes, I just added a type without containing the accountId and the amount.